### PR TITLE
UI Changes

### PR DESCRIPTION
**Pull request by prsws**
This version adds the following:

1) Equipment switch panel widget that displays the outlets as switches which not only depict their current state but also allows their switching on/off from the dashboard without having to go to the Equipment tab. It's available in addition to the original equipment bar chart.
Equipment Switch Panel

2) Displays equipment, timers, temperature, ATO, pH, dosing, macros and connectors **sorted alphanumerically by name** instead of by id.

3) Rounds pH graph Y-Axis, reading and tooltip values to 2 digits.

4) Additional Spanish translations .

### CODE REVIEW

**Comment by prsws** [comment]
Fixed translation problems after checks.

**Comment by ranjib** [comment]
@prsws thank you so much. Bless thy heart :-) . I so much wanted this feature. I can not wait to merge this in, reef-pi community will absolutely love this.

Few changes I need, if you don't mind:
- Exclude the README changes from that mentions this is a fork, since once merge it will be on main reef-pi
-  As of now we don't use changelog comments in individual files. If you don't mind please remove them, If you want to preserve this log, then please add a dedicated changelog file, I'm open to that. I personally just use release notes for top level changes and then dig down with git log if I need details. I would like to keep individual files pristine free from changelogs, unless there's some concern.

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@ coverage
 front-end/src/utils/translations/*.json
 test-data
 .vendor-new/*
+package-lock.json
```
Comment (ranjib) [inline .gitignore:25]: Dont exlcude this. We need this checked in and tracked for reproducible builds

**Comment by prsws** [comment]
> @prsws thank you so much. Bless thy heart :-) . I so much wanted this feature. I can not wait to merge this in, reef-pi community will absolutely love this.
> 
> Few changes I need, if you don't mind:
> 
> * Exclude the README changes
> * As of now we don't use changelog comments in individual files. If it's ok please remove them, If you want to preserve this log, we can add a changelog file. I personally just use release notes for top level changes and git log for details. I would like to keep individual files pristine unless there's some concern.

Ranjib:

I just made the requested changes. I would also like you to add the changelog file because I think it's useful. Thank you very much,

Regards,

José

File: .gitignore (hunk 1 above)
Comment (prsws) [reply .gitignore]: Done. Thank you.

Regards,

José

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,9 @@
 SOURCEDIR=.
 SOURCES = $(shell find $(SOURCEDIR) -name '*.go')
-VERSION=$(shell git describe --always --tags)
+VERSIONDATE:= $(shell date -Is)
+VERSION:=$(shell git describe --always --tags)
+VERSION:=$(addsuffix -MODIFIED, $(VERSION))
+#VERSION:=$(addsuffix $(VERSIONDATE), $(VERSION))
```
Comment (hectorespert) [inline Makefile:6]: @ranjib This change is required?

Comment (hectorespert) [reply Makefile:6]: @ranjib @prsws What are the purpose of this binary files?

Screenshot from 2021-01-31 11-33-10

Comment (prsws) [reply Makefile:6]: Hector:

This is my first time contributing to an open source project, so please excuse my newbie mistakes. I use the Makefile VERSION statements to identify my version as different from upstream when I post it to other forums such as Robo-Tank. In addition, the binary files are artifacts present in my development environment and were included in the merge unintentionally. Please let me know what else I need to clean up from my branch "uichanges2upstream" to make it proper for inclusion in the project. Thank you.

File: front-end/src/temperature/edit_temperature.jsx [hunk 3]
```diff
@@ -431,26 +431,6 @@ const EditTemperature = ({
             </div>
           </div>
         </div>
-        <div className='row'>
```
Comment (chrisleavoy) [inline front-end/src/temperature/edit_temperature.jsx:434]: @ranjib  @prsws was removing the hysteresis input here intended?

Comment (ranjib) [reply front-end/src/temperature/edit_temperature.jsx:434]: No

**Pull request merged**